### PR TITLE
add `doc/chap*` to `.gitignore`

### PR DESCRIPTION
**Pull request by ThomasBreuer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ./doc/chap*
 ./examples/output/*
 ./tst/output/*
+./doc/chap*
```
Comment (fingolfin) [inline .gitignore:4]: Isn't this the same as line 1?

Comment (ThomasBreuer) [reply .gitignore:4]: Strange, the generated files were listed as untracked as soon as I had processed the manual, and they disappear from the list of untracked files after my addition.
Even more strange, now the generated files are again listed as untracked; they disappear from this list when I remove the leading `./` from the lines in `.gitignore` --I do not understand what is going on here.

Comment (fingolfin) [reply .gitignore:4]: The problem is that the .gitignore is simply wrong. Fixed in PR #11

Comment (ThomasBreuer) [reply .gitignore:4]: Thanks. The format from #11 works for me.

**Pull request closed without merging**